model-optimizer/extensions/middle/SplitConcatPairToInterpolate.py

```diff
@@ -53,6 +53,9 @@ def get_concat_after_split(split: Node) -> Optional[Node]:
 
 
 def get_interpolate_pattern(split: Node) -> dict:
+    split_shape = split.in_port(0).data.get_shape()
+    if split_shape is None or len(split_shape) < 4:
```

Comment:
What if input rank is equal to 6 or 9? Let's limit it by saying:
```suggestion
    if split_shape is None or len(split_shape) < 4 or len(split_shape) > 5:
```

Reply:
Fixed.